docs/source/reference/config-user-env.md

```diff
@@ -24,51 +21,41 @@ This section will focus on user environments, including:
 
 ## Installing packages
 
-To make packages available to users, you generally will install packages
-system-wide or in a shared environment.
+To make packages available to users, you will typically install packages system-wide or in a shared environment.
 
-This installation location should always be in the same environment that
+This installation location should always be in the same environment where
 `jupyterhub-singleuser` itself is installed in, and must be _readable and
-executable_ by your users. If you want users to be able to install additional
+executable_ by your users. If you want your users to be able to install additional
 packages, the installation location must also be _writable_ by your users.
 
-If you are using a standard system Python install, you would use:
+If you are using a standard Python installation on your system, use the following command:
 
 ```bash
 sudo python3 -m pip install numpy
 ```
 
-to install the numpy package in the default system Python 3 environment
+to install the numpy package in the default Python 3 environment on your system
 (typically `/usr/local`).
 
-You may also use conda to install packages. If you do, you should make sure
-that the conda environment has appropriate permissions for users to be able to
-run Python code in the env.
+Alternatively, You may also use conda to install packages. To do this, ensure that the conda environment has appropriate users permissions needed to run Python code in the environment.
 
 ## Configuring Jupyter and IPython
 
 [Jupyter](https://jupyter-notebook.readthedocs.io/en/stable/config_overview.html)
 and [IPython](https://ipython.readthedocs.io/en/stable/development/config.html)
 have their own configuration systems.
 
-As a JupyterHub administrator, you will typically want to install and configure
-environments for all JupyterHub users. For example, you wish for each student in
-a class to have the same user environment configuration.
-
-Jupyter and IPython support **"system-wide"** locations for configuration, which
-is the logical place to put global configuration that you want to affect all
-users. It's generally more efficient to configure user environments "system-wide",
-and it's a good idea to avoid creating files in users' home directories.
+As a JupyterHub administrator, you will typically want to install and configure environments for all JupyterHub users. For example, let's say you wish for each student in a class to have the same user environment configuration.
 
+Jupyter and IPython support **"system-wide"** locations for configuration, which is the logical place to put global configuration that you want to affect all users. It's generally more efficient to configure user environments "system-wide",and it's a good practice to avoid creating files in the users' home directories.
```

Comment:
```suggestion
Jupyter and IPython support **"system-wide"** locations for configuration, which is the logical place to put global configuration that you want to affect all users. It's generally more efficient to configure user environments "system-wide", and it's a good practice to avoid creating files in the users' home directories.
```